code-challenges.test.js

```diff
@@ -26,26 +28,110 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+// const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
+
+describe("isDivisibleByThree", () => {
+    it("should return 15 is divisible by three when the input number is 15 or ", () => {
+      const object1 = { number: 15 }
+      const result = isDivisibleByThree(object1.number)
+      expect(result).toBe("15 is divisible by three")
+    })
+// })
+// Returns: ReferenceError: isDivisibleByThree is not defined
+
+// const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
+    it("should return 0 is divisible by three when the input number is 0", () => {
+        const object2 = { number: 0 }
+        const result2 = isDivisibleByThree(object2.number)
+        expect(result2).toBe("0 is divisible by three")
+    })
+// Returns: ReferenceError: isDivisibleByThree is not defined
+
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
+    it("should return -7 is not divisible by three when the input number is 7", () => {
+        const object3 = { number: -7 }
+        const result3 = isDivisibleByThree(object3.number)
+        expect(result3).toBe("-7 is not divisible by three")
+    })
+})
```

Comment:
Best practice would be to only have 1 describe and 1 it statement for a single function. Then you can have multiple expect statements, respective to how many test variables there are.